README.md

```diff
@@ -37,13 +37,21 @@ Make predictions
 model.predict(row_index, column_index);
 ```
 
-Get the latent factors (these approximate the training matrix)
+Get the latent factors of a specific row or column
+
+```rust
+model.p_row(row_index);
+model.q_col(column_index);
+```
+
+Get all the latent factors (these approximate the training matrix)
 
 ```rust
 model.p_factors();
 model.q_factors();
 ```
 
+
```

Comment:
Please remove / keep the changeset minimal.

Reply:
It's been fixed and put into one commit